src/EventPage.js

```diff
@@ -77,4 +87,10 @@ export default function EventPage() {
       )}
     </>
   )
+  function toggleHide(id) {
+    const index = eventArray.findIndex((event) => event.id === id)
+    const eventUpdate = eventArray[index]
```

Comment:
I think this might be shorter and also understandable

```suggestion
    const eventUpdate = eventArray.find((event) => event.id === id)
```